lib/nanoc/helpers/link_to.rb

```diff
@@ -88,8 +88,23 @@ def link_to_unless_current(text, target, attributes = {})
       path = target.is_a?(String) ? target : target.path
 
       if @item_rep && @item_rep.path == path
+        active_class = attributes[:active_class] || 'active'
+        attributes.delete :active_class
+
+        # merge active_class with attributes' class
+        if attributes.has_key? :class then
+          attributes[:class] += " #{active_class}"
+        else
+          attributes[:class] = active_class
+        end
+
+        # Join attributes, prefixed by a space
+        attributes = attributes.reduce('') do |memo, (key, value)|
```

Comment:
`#reduce` will be slow. Prefer appending to a string rather than creating new strings.